src/ui/frontend/src/views/Sites/Overview.vue

```diff
@@ -72,19 +103,35 @@ function hydrateForm() {
       vCenterServers: [...site.value.vCenterServers],
       aosList: [...site.value.aosList],
       hypervisorList: [...site.value.hypervisorList],
+      lcmDarksiteUrl: site.value.lcmDarksiteUrl,
+      prism: site.value.prism || form.site.prism,
+      smtp: site.value.smtp,
+      ldap: site.value.ldap
     }
+    form.smtp.templateString = site.value.smtp.fromTemplate
+    form.smtp.domain = site.value.smtp.fromDomain
   }
   return form
 }
 
+const certsProvided = computed(() => {
+  return site.value.prism.certificate && site.value.prism.caChain
```

Comment:
This leads to slightly weird behavior where we show the green check mark as soon as the user provides cert/CA in the form before saving. Knowing that we're doing that to try to keep the key out of the API makes sense and this is probably going to get punted to a later TODO as it would require some other helper type functions but finding a way to fully reflect this status accurately would be the same as solving this problem and providing indicators for credentials for things like LDAP.

Reply:
Created issue #5 so we don't forget about it.